src/main/resources/MessagingDescriptor.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Where is there a definition of a what a messaging description is and what it is used for?

Reply:
Messaging descriptor is a file that describes what events the module publishes and to which events module is subscribed. This file is used for registration of module to mod-pubsub by the method PubSubClientUtils#registerModule(). Please, find a definition of MessagingDescriptor file in the MessagingDescriptor section of [mod-pubsub documentation](https://github.com/folio-org/mod-pubsub#messaging-descriptor).

Reply:
Ok, thanks. I don't really understand why a file is required for this.